crates/test-utils/src/chain/test_chain.rs

```diff
@@ -1016,19 +1023,35 @@ impl TestChain {
             logs[0].clone().hyperdrive
         };
 
-        // Transfer ownership of the base token, factory, vault, and lido to the
-        // admin address now that we're done minting tokens and updating the
-        // configuration.
+        // Add the 4626 Hyperdrive instance and the Lido Hyperdrive instance
+        // to the registry contract.
+        hyperdrive_registry
+            .set_hyperdrive_info(vault.address(), uint256!(1))
+            .send()
+            .await?;
+        hyperdrive_registry
+            .set_hyperdrive_info(lido.address(), uint256!(1))
+            .send()
+            .await?;
```

Comment:
Let's add some assertions in the test cases that verifies that the registry was properly updated.